test/decoders/test_video_decoder_ops.py

```diff
@@ -322,6 +323,54 @@ def test_frame_pts_equality(self):
             )
             assert pts_is_equal
 
+    @pytest.mark.parametrize("color_conversion_library", ("filtergraph", "swscale"))
+    def test_color_conversion_library(self, color_conversion_library):
+        decoder = create_from_file(str(NASA_VIDEO.path))
+        _add_video_stream(decoder, color_conversion_library=color_conversion_library)
+        frame0, _, _ = get_next_frame(decoder)
```

Comment:
`frame0, *_ = get_next_frame(decoder)` will also work, and is more future-proof.

Reply:
Great idea. Done.